src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -9,7 +15,36 @@ void EnDns_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnDns_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnDns_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void EnDns_ChangeAnim(EnDns* this, u8 arg1);
+
```

Comment:
this forward declaration shouldnt be needed, the function is defined above its first usage